docker-compose.yaml

```diff
@@ -1,4 +1,7 @@
 version: "3.9"
+networks:
```

Comment:
I think it's better to remove this until we are trying to connect to the sequencer nodes started from the espresso sequencer repo.